app/code/Magento/Captcha/composer.json

```diff
@@ -13,7 +13,7 @@
         "zendframework/zend-session": "^2.7.3"
     },
     "type": "magento2-module",
-    "version": "100.2.0",
+    "version": "100.2.1",
```

Comment:
please, revert this fix. We will update the package versions on our side when needed